flocker/node/_deploy.py

```diff
@@ -473,6 +475,7 @@ def applications_from_units(result):
                 not_running=not_running,
                 used_ports=self.network.enumerate_used_ports(),
                 manifestations=manifestations,
+                paths=manifestation_paths,
```

Comment:
Because this isn't something you can configure, and we want `Manifestation` etc. to be only stuff shared by both configuration and state. I'll right it down.